denonavr/const.py

```diff
@@ -261,7 +261,7 @@
     command_play=COMMAND_PLAY)
 
 # Telnet Commands
-TELNET_EVENTS = ["PW", "MV", "MU", "SI", "MS", "PS"]
+TELNET_EVENTS = ["HD", "MS", "MU", "MV", "NS", "PS", "PW", "SI", "SS", "TF"]
```

Comment:
If this is being used for 'in' calls it might make sense to make it a set to avoid linear searches